app/assets/javascripts/backbone/views/loan_charts_view.coffee

```diff
@@ -52,18 +58,27 @@ class MS.Views.LoanChartsView extends Backbone.View
     chart.draw(data, options)
 
   breakevenProductProfitChart: ->
-    data = new google.visualization.DataTable();
+    data = new google.visualization.DataTable()
     data.addColumn 'string', I18n.t('loan.breakeven.product')
     data.addColumn 'number', I18n.t('loan.breakeven.profit')
+    options = @defaultChartOptions
 
+    negativeNums = false
```

Comment:
There is a possibility that this chart may be negative if cost to produce is more than revenue for products. I've displayed a bar chart if that happens. The default Google Chart error message in red has impact on charts surrounding it if values are negative. Google Charts cannot display negative values in a pie chart.